### DISCUSSION TITLE
Cross-PDF Library Discussion Channel

### DISCUSSION THREAD

**Discussion by MartinThoma** [pymupdf/PyMuPDF#2206] [Looking for help]
Hey PyMuPDF devs!

I'm Martin, the maintainer of pypdf (formerly pyPdf) and PyPDF2 (now merged back into pypdf) :wave: 

Would anybody here be open to participate in discussions about PDF which are not related to pdfminer?

I think it could be mutually beneficial to have an easy way to get feedback :-)
We could use this as a communication channel: https://github.com/py-pdf/pypdf/discussions/categories/pdf-format-questions

Concretely, I would be curious how you do this: https://github.com/py-pdf/pypdf/discussions/1594

**Comment by jamie-lemon** [comment]
Hi Martin,
Thanks for reaching out! Generally we use our Discord channel for in-depth discussions now. Feel free to join that and we can discuss these kind of things there!

Have a good day :)

**Reply by MartinThoma** [reply]
How do I find your discord channel? (I've never used Discord before)

**Reply by JorjMcKie** [reply]
Here is a [link](https://github.com/pymupdf/PyMuPDF/discussions/1766)
